dashboard/package.json

```diff
@@ -62,10 +65,10 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.5",
     "redux-thunk": "^2.2.0",
+    "seamless-immutable": "^7.1.4",
     "semver": "^5.6.0",
-    "ts-jest": "^22.0.1",
```

Comment:
I wasn't sure why this was in our dependencies (since it should only be for devDependencies below right?).